openstef/model/serializer.py

```diff
@@ -300,6 +305,55 @@ def _get_feature_names(
                 )
         return model_specs.feature_names
 
+    def _get_feature_modules(
```

Comment:
Nice addition and nice exception handling with clear messagages! As there is some logic in this functions, could you please make some unit tests that check the if statements in lines 342 and 344?

Reply:
The tests has been added. Coverage is 100% now. Thank you.